unison-src/transcripts/error-messages.output.md

```diff
@@ -343,10 +343,12 @@ use.keyword.in.namespace = 1
 
   Loading changes detected in scratch.u.
 
-  The identifier used here isn't allowed to be a reserved keyword: 
+  The identifier, `namespace`, used here is a reserved keyword: 
```

Comment:
I think breaking up "The identifier used here" with commas is weird / possibly a mistake.

Maybe:

> The identifier `namespace` used here is a reserved keyword

or

> The identifier used here, `namespace`, is a reserved keyword

While we're in here, could you add cases in the transcript for the ones that didn't have coverage, so we can scrutinize their commas too? 🙏 
`ReservedSymbolyId`, `InvalidSymbolyId`, `InvalidWordyId`.

Reply:
> I think breaking up "The identifier used here" with commas is weird / possibly a mistake.

You’re right – I thought appositives always needed to be offset with commas, but not if there is already some other punctuation (the backticks in this case).

> While we're in here, could you add cases in the transcript for the ones that didn't have coverage, so we can scrutinize their commas too? 🙏 `ReservedSymbolyId`, `InvalidSymbolyId`, `InvalidWordyId`.

`InvalidWordyId` isn’t even used, so I removed that case. The other two I’m not sure are reachable. E.g., [the transcript case that is ostensibly a `ReservedSymbolyId` error actually fails differently](https://github.com/unisonweb/unison/pull/5108/files#diff-00211f24524c732d516cdbe237e28deb0c74f20efe6f87ef55e2ceeb32e973abR355-R374).

Reply:
I see.  Another variant gave another error:

```
(!) a b = 1
```
```
  I got confused here:
  
      1 | (!) a b = 1
  
  
  I was surprised to find a ( here.
  I was expecting one of these instead:
  
  * ability
  * newline or semicolon
  * type
  * use
```
Not sure what's going on there.

Reply:
(whoops this was with the wrong verson)

Reply:
Disregard, it looks the same in this branch.